HealthVault/Classes/Caching/Private/MHVThingCacheProtocol.h

```diff
@@ -90,6 +90,18 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)cacheMethod:(MHVMethod *)method completion:(void (^)(MHVPendingMethod *_Nullable pendingMethod, NSError *_Nullable error))completion;
 
 /**
+ Adds 'placeholder' things to the cache for a recordId.
+ @note A 'placeholder' thing is a thing that was created while offline. Once the internet connection is restored the original thing creation request will be issued to the cloud and the 'placeholder' will be replaced with the HealthVault version of the thing. 'placeholder' Things do not have a Thing Key property set.
+ 
```

Comment:
That's might be true now, but we need to support updating, and deleting 'placeholder' things. We might need to add a temporary key to support this, but this missing behavior is a bug until we can fix it.